src/compositions/templates/case-study/Success/styled.js

```diff
@@ -0,0 +1,33 @@
+import styled from 'styled-components';
+import { textLighter, textLight } from 'styles/colours';
+
+export const Title = styled.h2`
+  font-size: 36px;
+  font-weight: 500;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.61;
+  letter-spacing: normal;
+  color: ${textLight};
+`;
+
+export const Main = styled.p`
+  p-style-type: none;
+  font-size: 24px;
+  font-weight: 300;
+  font-style: normal:
```

Comment:
Same here, remove the unneeded properties `font-style` `font-stretch` `letter-spacing`.

Try it without the line height first and see if the global line height looks close to the design.